app/models/user.rb

```diff
@@ -1,32 +1,28 @@
 class User < ApplicationRecord
-  has_many :followed_users, foreign_key: :follower_id, class_name: 'Friendship'
+  has_many  :followed_users,
+            foreign_key: :follower_id,
+            class_name: 'Friendship',
+            dependent: :destroy
   has_many :followees, through: :followed_users
-  has_many :following_users, foreign_key: :followee_id, class_name: 'Friendship'
+  has_many  :following_users,
+            foreign_key: :followee_id,
+            class_name: 'Friendship',
+            dependent: :destroy
```

Comment:
This feels way more readable than the single line.

Reply:
YEAH! Another amazing rubocop catch, it caps you at like 140 characters or something :)